rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/providers/CloudProviderBakeHandler.groovy

```diff
@@ -140,7 +141,21 @@ abstract class CloudProviderBakeHandler {
     def virtualizationSettings = findVirtualizationSettings(region, bakeRequest)
 
     BakeOptions.Selected selectedOptions = new BakeOptions.Selected(baseImage: findBaseImage(bakeRequest))
-    def (imageName, appVersionStr, packagesParameter) = imageNameFactory.deriveImageNameAndAppVersion(bakeRequest, selectedOptions)
+
+    List<String> packageNameList = bakeRequest?.package_name?.tokenize(" ")
+    def appVersionStr
+
+    def osPackages = packageNameList.collect { name ->
+      PackageNameConverter.buildOsPackageName(selectedOptions.baseImage.packageType, name)
+    }
+
+    if (!osPackages.empty) {
+     appVersionStr = imageNameFactory.buildAppVersionStr(bakeRequest, osPackages.first())
```

Comment:
Any reason not to make `imageNameFactory.buildAppVersionStr()` take the full list of packages, instead of extracting the first package here at the call-site, so it is more consistent with its siblings?